### DISCUSSION TITLE
Visible from the control

### DISCUSSION THREAD

**Discussion by Mephisto090** [hardkoded/puppeteer-sharp#1992] [Q&A] [answered]
Hi, in which file is hidden that buttons that are pressed on a website must be visible from the control? I would like to change that for myself so that buttons can also be pressed that are not visible to the user?

(Node is either not visible or not an HTMLElement)

regards

**Comment by kblok** [comment, marked as answer]
That's on the browser side

**Reply by Mephisto090** [reply]
That was fast. :D Can you tell me which file exactly?

**Reply by kblok** [reply]
On the browser I mean chromium. I have no idea where that is :p

**Reply by Mephisto090** [reply]
Ah ok thanks
